user/utils.py

```diff
@@ -0,0 +1,20 @@
+import jwt
+from django.http import request,JsonResponse
+from my_settings import SECRET, ALGORITHMS
+from user.models import User
+
+def check_user(func):
+    def wrapper_func(self, request):
+        for_client_token = request.headers.get('authorization',None)
```

Comment:
dictionary에 'authorization'이라는 key 값으로 접근할 때, 없는 경우 default로 None을 반환합니다.
그렇기 때문에, 두번째 인자로 None을 전달해줄 필요가 없습니다.
```suggestion
        access_token = request.headers.get('authorization')
```

Reply:
변수명은 명사로 작성해주세요!

Reply:
token의 종류가 많을 수 있습니다.
access_token? refresh_token? csrf_token?
그렇기 때문에 명확하게 어떤 token인지 명시해주셔야 됩니다!

Reply:
네, 변수명 수정했습니다~!